Formula/apache-flex-sdk.rb

```diff
@@ -0,0 +1,49 @@
+class ApacheFlexSdk < Formula
+  desc "Framework for building expressive web and mobile applications"
+  homepage "https://flex.apache.org/"
+  url "http://mirror.fibergrid.in/apache/flex/installer/3.2/apache-flex-sdk-installer-3.2.0-src.tar.gz"
```

Comment:
I just realized that the source indicated is the installer, not the source or the binary.
The URL should be from here: https://flex.apache.org/download-source.html
or here: https://flex.apache.org/download-binaries.html
Version: 4.16